graph/resolver/schema.resolvers.go

```diff
@@ -489,6 +489,36 @@ func (r *mutationResolver) InviteUserToDiscussion(ctx context.Context, discussio
 	return r.DAOManager.InviteUserToDiscussion(ctx, userID, discussionID, invitingParticipantID)
 }
 
+func (r *mutationResolver) InviteTwitterUsersToDiscussion(ctx context.Context, discussionID string, twitterUsers []*model.TwitterUserInput, invitingParticipantID string) ([]*model.DiscussionInvite, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return nil, fmt.Errorf("Need auth")
+	}
+
+	participantResponse, err := r.DAOManager.GetParticipantsByDiscussionIDUserID(ctx, discussionID, authedUser.UserID)
+	if err != nil {
+		return nil, err
+	}
+	if participantResponse == nil {
+		return nil, fmt.Errorf("Failed to find participant with ID %s", invitingParticipantID)
+	}
+
+	// Verify that the inviting participant belongs to the logged-in user and that they're not in incognito
```

Comment:
This implements a sort of access control. The desired semantics is:
- Only public-identity users can invite other users
- A user should not "hack" the system and invite people on behalf of other users. This could leak anonymity infos.